pim-tool-back/src/main/java/pilotproject/Project_Management_ELCA/model/exception/ProjectNumberExistedException.java

```diff
@@ -0,0 +1,7 @@
+package pilotproject.Project_Management_ELCA.model.exception;
+
+public class ProjectNumberExistedException extends RuntimeException {
+    public ProjectNumberExistedException(String message) {
+        super(message);
```

Comment:
[C] Exception should leave a stack trace with enough information for Developer to debug, such as: What kind of exception? What information, project number that already existed?